ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -60,11 +60,114 @@ namespace tensorflow {
 
 namespace ngraph_bridge {
 
+int NGraphEncapsulateOp::s_instance_id = 0;
+
 //---------------------------------------------------------------------------
 //  NGraphEncapsulateOp::ctor
 //---------------------------------------------------------------------------
 NGraphEncapsulateOp::NGraphEncapsulateOp(OpKernelConstruction* ctx)
     : OpKernel(ctx) {
+  // Set the backend type for the this NGraphEncapsulate Op
+  std::string backend_name;
+  OP_REQUIRES_OK(ctx, ctx->GetAttr<string>("ngraph_backend", &backend_name));
+  std::string device_id;
+  OP_REQUIRES_OK(ctx, ctx->GetAttr<string>("ngraph_device_id", &device_id));
+
+  // Concatenate the backend_name:device_id
+  string be_name;
+  try {
+    be_name = BackendManager::GetBackendCreationString(backend_name, device_id);
```

Comment:
BackendManager::GetBackendCreationString(backend_name, device_id) will not throw an error
so we dont need a try catch here
https://github.com/tensorflow/ngraph-bridge/blob/master/ngraph_bridge/ngraph_backend_manager.cc#L195